analyzers/tests/SonarAnalyzer.UnitTest/CFG/Roslyn/BasicBlockTest.cs

```diff
@@ -118,6 +124,88 @@ public void Method(bool condition)
             branch.Predecessors.Should().HaveCount(1);
             assign.Predecessors.Should().HaveCount(1);
             exit.Predecessors.Should().HaveCount(2);
+
+            entry.SuccessorBlocks.Should().HaveCount(1).And.Subject.Single().Should().Be(branch);
+            branch.SuccessorBlocks.Should().HaveCount(2).And.Subject.Should().ContainInOrder(assign, exit);
+            assign.SuccessorBlocks.Should().HaveCount(1).And.Subject.Single().Should().Be(exit);
+            exit.SuccessorBlocks.Should().HaveCount(0);
+        }
+
+        [TestMethod]
+        public void ValidateOperations()
+        {
+            const string code = @"
+public class Sample
+{
+    int Pow(int num, int exponent)
+    {
+        num = num * Pow(num, exponent - 1);
+        return 42;
+    }
+}";
+            var cfg = TestHelper.CompileCfg(code);
+            var entry = cfg.EntryBlock;
+            var exit = cfg.ExitBlock;
+            var body = cfg.Blocks[1];
```

Comment:
_Nitpick:_ I'd swap the lines to have a logical order.
Same for the assertions below.
```suggestion
            var body = cfg.Blocks[1];
            var exit = cfg.ExitBlock;

```

Reply:
>Same for the assertions below.